src/App.js

```diff
@@ -146,7 +91,81 @@ const App = () => {
 
   async function onFileChange(event) {
     const url = URL.createObjectURL(event.target.files[0]);
-    wvLoadVideo(url);
+
+    // Can also detect by specific video file types (ie. mp4, ogg, etc.)
+    if (event.target.files[0].type.includes('video')) {
+      initializeWebViewerSettings(wvInstance);
+      wvLoadVideo(url);
+      setTimeout(() => {
```

Comment:
Notes panel wouldn't open unless a small time out was added, probably due to webviewer instance resolving video.

Reply:
Why does it need the timeout on the first load but it does when you switch?

Reply:
not 100% sure here, I was trying to do some debugging on the lifecycle of webviewer, but went down the rabbit hole for a few hours :)